_posts/2023-05-24-windows-setup.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: "5.24_NOTES_AnjaliN"
```

Comment:
This title will appear at the top of the post (and on the main page), so it should be something a bit more descriptive.

Probably something more like "ARCS Lab Windows PC Setup"